src/app/multicurrency-location-pick/multicurrency-location-pick.component.scss

```diff
@@ -0,0 +1,26 @@
+$colour-multicurrency-brand: #ed6743;
+$colour-multicurrency-brand-darker: #b14324;
```

Comment:
I think we should probably avoid adding new colours linked to the brand unless there's a reference using these @alihejazi? 

https://gogiveone.org/ does have a slightly different hover colour but it uses #f24d2d

Reply:
Cool, I've removed the darker colour.